### PR TITLE
the address of a variable cannot be nil

### LINKED ISSUE #1147: the address of a variable cannot be nil
https://github.com/kedacore/keda/blob/6dfe7f7c389090378bfea4fe94f953f7f11df7f7/pkg/scaling/resolver/scale_resolvers.go#L23-L34

the `container` can not be nil

### PR DESCRIPTION
**Pull request by silenceper**
Signed-off-by: silenceper <silenceper@gmail.com>

_Provide a description of what has been changed_

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)
- [ ] Tests have been added
- [ ] A PR is opened to update the documentation on https://github.com/kedacore/keda-docs
- [ ] Changelog has been updated

Fixes #1147

### CODE REVIEW

File: pkg/scaling/resolver/scale_resolvers.go [hunk 1]
```diff
@@ -22,14 +22,16 @@ func ResolveContainerEnv(client client.Client, logger logr.Logger, podSpec *core
 
 	var container corev1.Container
 	if containerName != "" {
+		findMathContainer := false
```
Comment (zroubalik) [inline pkg/scaling/resolver/scale_resolvers.go:25]: there's probably typo in the variable name, eg. `containerWithNameFound` or something like that

**Review by zroubalik — CHANGES_REQUESTED** [review]
Good catch! minor nit on the naming

File: pkg/scaling/resolver/scale_resolvers.go (hunk 1 above)
Comment (silenceper) [reply pkg/scaling/resolver/scale_resolvers.go]: fixed. `containerWithNameFound` would be better.

**Review by zroubalik — APPROVED** [review]
LGTM, thanks!

**Pull request merged**